rllib/algorithms/ppo/ppo_rl_module_config.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+
+import gymnasium as gym
+
+from ray.rllib.core.models.catalog import Catalog
+from ray.rllib.core.rl_module.rl_module import RLModuleConfig
+from ray.rllib.utils.annotations import ExperimentalAPI
+
+
+@ExperimentalAPI
+@dataclass
+class PPOModuleConfig(RLModuleConfig):
+    """Configuration for the PPORLModule.
+
+    Attributes:
+        observation_space: The observation space of the environment.
+        action_space: The action space of the environment.
+        catalog: The PPOCatalog object to use for building the models.
+        free_log_std: For DiagGaussian action distributions, make the second half of
```

Comment:
I think this parameter should become part of model_config_dict. We should not keep carrying it around.

Reply:
Yes, also to this day we assert it's False when building. Thanks!